docs/_static/css/custom-theme.css

```diff
@@ -270,10 +203,31 @@ footer a {
 
 /* Make method links like :meth:`~.get_data_points` not tiny */
 code, .rst-content tt, .rst-content code {
-    font-size: 100% !important;
+  font-size: 90% !important;
+  font-weight: 500 !important;
 }
 
-/* hide branches from version switcher */
-.rst-other-versions dl:last-child {
-    display: none;
-}
\ No newline at end of file
+a > code {
+  text-decoration: underline;
+}
+
+.rst-content code.xref {
+  color: #e31b37;
+}
+
+.rst-content a {
+  font-size: inherit;
```

Comment:
do we need font-size:inherit?

Reply:
yes this is for links within other context ie. headers, lists, paragraphs. we want them to inherit font size of their context